python/semantic_kernel/connectors/ai/open_ai/services/open_ai_chat_completion_base.py

```diff
@@ -65,6 +68,7 @@ def get_prompt_execution_settings_class(self) -> "PromptExecutionSettings":
         """Create a request settings object."""
         return OpenAIChatPromptExecutionSettings
 
+    @trace_chat_completion(MODEL_PROVIDER_NAME)
```

Comment:
We want a decorator there too, yes.  It might be a bit different, though.  For now, I'm just trying to have the smallest PR possible since it's already a hurdle for me to get the current one in!

I intend to have smaller PR's for streaming and for text completions.

Reply:
I actually added one for text completions.  Streaming completions to come.